library/mongo-express

```diff
@@ -2,7 +2,13 @@
 
 Maintainers: Nick Cox <nickcox1008@gmail.com> (@knickers)
 GitRepo: https://github.com/mongo-express/mongo-express-docker.git
-GitCommit: 4b43fe8a1206434cb32a006cd155dd71462f092f
 
+
+GitCommit: 26e7ea94e4d222de7d52531caee52149ac093c26
+Tags: 1.0.0-alpha.4, 1.0.0-alpha, 1.0, latest
```

Comment:
Duplicate `latest` tag needs to be removed from here or from `0.54` tags below.

Reply:
My bad, done